docs/add-ins/addin-tutorial.md

```diff
@@ -1,107 +1,222 @@
 
-## Write the code
+    ```bash
+    cd "Git the gist"
+    ```
+
+1. This add-in will use the [Showdown](https://github.com/showdownjs/showdown) library to convert Markdown to HTML and the [URI.js](https://github.com/medialize/URI.js) library to build relative URLs. To install these tools for your project, run the following command in the root directory of the project:
+
+    ```bash
+    npm install showdown urijs --save
+    ```
+
+### Update the manifest
+
+The manifest for an add-in controls how it appears in Outlook. It defines the way the add-in appears in the add-in list and the buttons that appear on the ribbon, and it sets the URLs for the HTML and JavaScript files used by the add-in. 
+
+#### Specify a support page
+
+The manifest the generator creates contains a placeholder value for the `SupportUrl` element that's not a valid URL. To prevent the file from failing validation, complete the following steps:
+
+1. In the root directory of the project, create a new file named **support.html** and add the following markup.
 
-Now that we've verified that the base add-in works, we can customize it to do what we want. We'll start with the manifest.
+    ```html
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, inline-scale=1">
+        <title>Support</title>
+      </head>
+      <body>
+        <h1>Support</h1>
+        <p>This is the support page for the <b>Git the gist</b> add-in.</p>
+        <p>In a real-world scenario, this page should provide support information for the add-in.</p>
+      </body>
+    </html>
+    ````
 
-### Update the manifest (again)
+1. Open the **manifest.xml** file and update the `SupportUrl` element to point to the **support.html** file that you created.
 
-The manifest for an add-in controls how it appears in Outlook. It defines the way the add-in appears in the add-in list, the buttons that appear on the ribbon, and it sets the URLs for the HTML and JavaScript files used by the add-in.
+  ```xml
+  <SupportUrl DefaultValue="https://localhost:3000/support.html" />
+  ```
 
-Let's start by updating some properties of the add-in itself.
+#### Specify basic information
 
-1. Open the `manifest.xml` file. Locate the `ProviderName` element in the XML and replace the default value with your company name.
+Next, make the following updates in the **manifest.xml** file to specify some basic information about the add-in:
+
+1. Locate the `ProviderName` element and replace the default value with your company name.
 
    ```xml
     <ProviderName>Contoso</ProviderName>
     ```
-1. Update the `Description` with a description of the add-in.
+1. Locate the `Description` element, replace the default value with a description of the add-in, and save the file.
 
     ```xml
-    <Description DefaultValue="Allows users to access their Gists on GitHub"/>
+    <Description DefaultValue="Allows users to access their gists on GitHub"/>
```

Comment:
nit: maybe "...gists from GitHub" or "their GitHub gists"

Reply:
Will make this change in my new PR (coming momentarily).